### PR TITLE
Add feature/teletubbies

### PR DESCRIPTION
**Pull request by whitecollin**
- add: teletubbies title
- add: teletubbies list
- add: teletubbies search
- add: scroll event

### CODE REVIEW

File: src/pages/Teletubbies/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      {teletubbies.slice(0 , 20+offset)
+        .filter(list=>list.name.toLowerCase().includes(searchWord))
+        .map((item, key) => {
+          return (
+            <Teletubbie
+              way={key % 2}
+              key={`teletubbie_${key}`}
+              name={item.name}
+              description={item.description}
+              traits={item.traits} />
```
Comment (codeLover1126) [inline src/pages/Teletubbies/index.js:92]: Please place `/>` into the next line and check other parts as well.

File: src/components/Teletubbie/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <Box sx={{p: '10px'}}>
+        <Typography variant='h3' align='left'>
+          {name}
+        </Typography>
+        <Typography variant='body1' align='left' mb='2'>
+          {description}
+        </Typography>
+        <Typography marginTop={2} align='left'>
+          <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: '20px'}}>
+            {traits.map((item, key) => {
```
Comment (codeLover1126) [inline src/components/Teletubbie/index.js:27]: here you can make it simple by omitting `{`, `}`, and `return`. please take advantage of arrow function in ECMA script.

File: src/pages/Teletubbies/index.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      })()
+    }
+    catch(err){
+      setAlert(true);
+    }
+  }, []);
+  
+  useEffect(() => {
+    const handleScroll = () => {
+      if (
```
Comment (codeLover1126) [inline src/pages/Teletubbies/index.js:34]: you can make it one line, right?

**Review by codeLover1126 — CHANGES_REQUESTED** [review]
please check comments

File: src/pages/Teletubbies/index.js (hunk 1 above)
Comment (whitecollin) [reply src/pages/Teletubbies/index.js:92]: fixed it

File: src/components/Teletubbie/index.js (hunk 2 above)
Comment (whitecollin) [reply src/components/Teletubbie/index.js:27]: fixed it

File: src/pages/Teletubbies/index.js (hunk 3 above)
Comment (whitecollin) [reply src/pages/Teletubbies/index.js:34]: fixed it

File: src/pages/Teletubbies/index.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      <Snackbar
+        open={alert}
+        autoHideDuration={6000}
+        onClose={handleClose}
+        message="Fail in load"
+        action={show}
+      />
+      {teletubbies.slice(0 , 20+offset)
+        .filter(list=>list.name.toLowerCase().includes(searchWord))
+        .map((item, key) => {
```
Comment (codeLover1126) [inline src/pages/Teletubbies/index.js:82]: can you please check this as well?

**Review by codeLover1126 — CHANGES_REQUESTED** [review]
Check comments again.

File: src/pages/Teletubbies/index.js (hunk 4 above)
Comment (whitecollin) [reply src/pages/Teletubbies/index.js]: fixed it

**Pull request merged**